discovery-provider/plugins/network-monitoring/src/db/index.ts

```diff
@@ -19,11 +19,16 @@ export const sequelizeConn = new Sequelize(
     port: db.port,
     dialect: 'postgres',
     logging: false,
+    retry: {
+      match: [/Deadlock/i],
+      max: 1,
+    },
     pool: {
       max: dbConnectionPoolMax,
       min: 5,
-      acquire: 60000,
-      idle: 10000
+      // @note https://github.com/sequelize/sequelize/issues/8133#issuecomment-359993057
```

Comment:
fine for now but we should really make the correct fix soon - per your linked comment, the real fix is to address long running queries, which there should be none of